.prettierignore

```diff
@@ -0,0 +1,4 @@
+jest-reports
+README.md
+*.njk
```

Comment:
Perhaps one for a future PR but we may be able to add.njk files as an override.  Something like:
https://github.com/prettier/prettier-vscode/issues/1405

Reply:
Unfortunately, this condensed them into 1 line and there are also some errors. So definitely one for a future PR me thinks!